README.md

```diff
@@ -154,8 +154,8 @@ USAGE
   [--loglevel trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]
 
 OPTIONS
-  -d, --outputdir=outputdir                                                         [default: ~/.commerce
-                                                                                    /force-app] Directory to output the
+  -d, --outputdir=outputdir                                                         [default: ~/.commerc
+                                                                                    e/force-app] Directory to output the
```

Comment:
nit: format

Reply:
This file is automatically updated, the format always look weird in the diff but the README itself should be fine.